components/organisms/Modal.js

```diff
@@ -70,10 +70,10 @@ export const Modal = (props) => {
   return (
     <StyledModal show={props.show}>
       <StyledModalContent {...props}>
-      {props.closeButton && <StyledCloseButton onClick={props.onHideClick} icon={<MdClose/>} position={props.closeButton} />}
+      {props.closeButton && <StyledCloseButton onClick={props.onCloseClick} icon={<MdClose/>} position={props.closeButton} />}
         {React.Children.map(props.children, child => React.cloneElement(child), props)}
       </StyledModalContent>
-      {props.show && <StyledModalBackdrop onClick={props.onHideClick} />}
+      {props.show && <StyledModalBackdrop onClick={props.onCloseClick} />}
```

Comment:
It seems like in ooni/run we are using the prop `onHideClick`: https://github.com/ooni/run/blob/master/pages/index.js#L311. I don't have an opinion on if we should use the prop `onHideClick` or `onCloseClick` as long as when we cut a release with this we don't forget to update ooni/run to use the correct prop.

Reply:
Well, it looked like `onHideClick` was partially renamed to `onCloseClick`. If it is being used, then I will just update everything to `onHideClick` here.

Reply:
This is done.